server/src/main/java/org/opensearch/search/aggregations/support/CoreValuesSourceType.java

```diff
@@ -189,6 +193,101 @@ public DocValueFormat getFormatter(String format, ZoneId tz) {
             return DocValueFormat.GEOHASH;
         }
     },
+    GEO_SHAPE() {
```

Comment:
Should we call it GEOSHAPE similar to GEOPOINT ?

Reply:
I like GEO_SHAPE, as we use geo_shape. I think for GEOPOINT it could be a miss.